tests/test_tracer.py

```diff
@@ -18,8 +18,6 @@ def test_tracer_defaults(tracer_provider: InstanaTracerProvider) -> None:
         tracer_provider._propagators,
     )
 
-    assert tracer.tracer_id > INVALID_SPAN_ID
-    assert tracer.tracer_id <= _SPAN_ID_MAX_VALUE
```

Comment:
You must move this checking to the `test_tracer_create_span_context()` once there is the place the IDs are created now.

In addition, you must create a new `span_context` fixture in the `conftest.py` file where the `tracer_id` and `span_id` are equal, and then, create a new `test_tracer_create_span_context()`  function to test when they are equal.

Reply:
trace and span ids are equal only when the `parent_context` is None (i.e.) root span. So I've created a test case for the same.